taxonomy-manager-rest/src/main/java/com/digirati/taxman/rest/taxonomy/ProjectResource.java

```diff
@@ -3,17 +3,9 @@
 import com.digirati.taxman.common.rdf.annotation.jsonld.JsonLdFrame;
 import com.digirati.taxman.common.taxonomy.ProjectModel;
 import com.digirati.taxman.rest.MediaTypes;
-import com.digirati.taxman.rest.Roles;
 
-import javax.annotation.security.RolesAllowed;
 import javax.validation.Valid;
-import javax.ws.rs.BeanParam;
```

Comment:
Wildcard imports should be avoided. Would be better to revert back.

Reply:
I blame IntelliJ. Can revert.